src/components/SceneStats/SceneStats.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { formatTime } from 'lib/date'
+import Stats from './Stats'
+import { Props } from './SceneStats.types'
+
+export default class SceneStats extends React.PureComponent<Props> {
+  componentWillMount() {
+    this.fetchStats(this.props.deployment.base)
+  }
+
+  componentWillReceiveProps(nextProps: Props) {
+    if (nextProps.deployment.base !== this.props.deployment.base) {
+      this.fetchStats(nextProps.deployment.base)
+    }
+  }
+
+  fetchStats = (base: string) => {
+    const { stats, isLoading, onFetchStats } = this.props
+    if (!stats && !isLoading) {
+      onFetchStats(base)
+    }
+  }
+
+  render() {
+    const { stats, isLoading } = this.props
+    return (
+      <>
```

Comment:
This is not wrong but I'm a bit hesitant to return a component that spits out unwrapped elements cause we can't scope them (to style it for example)

You think that might be a problem at some point?

Reply:
mm since these are all `Stats`, if we wanted to style them from the inside we can just use `Stats.css`. If we wanted to style them from the outside, the parent can still wrap it and style it. But if I wrap it here, it's impossible to use it unwrapped from the outside, for instance to add these as siblings to other components and align them using flex, which is what I'm doing on the parent components. That's why I left it unwrapped.

Reply:
True, I guess you can just wrap them on the parent. What I was thinking is that if you do `.Parent .Stats` that could mean these or decentraland-ui's for example.
But not a problem really, just wanted to point that out just in case